With this patch, py3-sphinx_rtd_theme is not needed
as makedepends and can remain in the community repo


--- a/Documentation/conf.py
+++ b/Documentation/conf.py
@@ -35,7 +35,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'sphinx_rtd_theme'
+html_theme = 'default'
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
@@ -75,8 +75,6 @@
     ('mkfs.btrfs', 'mkfs.btrfs', 'create a btrfs filesystem', '', 8),
     ('btrfs-man5', 'btrfs', 'topics about the BTRFS filesystem (mount options, supported file attributes and other)', '', 5),
 ]
-
-extensions = [ 'sphinx_rtd_theme' ]
 
 # Cross reference with document and label
 # Syntax: :docref`Title <rawdocname:label>`
--- a/Documentation/requirements.txt
+++ /dev/null
@@ -1 +0,0 @@
-sphinx_rtd_theme
